src/components/WidgetButton/widget-button.jsx

```diff
@@ -0,0 +1,12 @@
+const WidgetButton = ({ label, handleClick }) => {
```

Comment:
Elsewhere I'm asking folks to use named exports instead of default exports; could you change this to `export const WidgetButton ...` and remove line 12 here?